types/lodash/array/zip.d.ts

```diff
@@ -76,7 +76,7 @@ declare module "../index" {
         zip<T>(
             this: LoDashImplicitWrapper<List<T> | null | undefined>,
             ...arrays: Array<List<T> | null | undefined>
-        ): LoDashImplicitWrapper<T[][]>;
+        ): LoDashImplicitWrapper<Array<Array<TResult | undefined>>>;
```

Comment:
Where did this `TResult` come from? Why didn't you amend the other overloads for the wrappers?

Reply:
That was a copy and paste mistake! Should have been `T`. Not sure why the build still succeeded. In any case, I've pushed a fix for this.

Reply:
I wrote here why it succeeded: #23307